### DISCUSSION TITLE
Types or code suggestions

### DISCUSSION THREAD

**Discussion by phantom-yak-venus** [buefy/buefy#4084] [Q&A] [answered]
Hi, everyone!

Is there a way to have code suggestions in vscode?

![image|300](https://github.com/user-attachments/assets/4e757c37-0ecb-40cd-b438-7a0af107de37)

I use buefy in my nuxt project as nuxt plugin. Here's the code of my buefy nuxt plugin:

```ts
import Buefy from 'buefy'
import 'buefy/dist/buefy.css'

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(Buefy, {})
})
```

**Comment by wesdevpro** [comment]
> Hi, everyone!
> 
> 
> 
> Is there a way to have code suggestions in vscode?
> 
> 
> 
> ![image|300](https://github.com/user-attachments/assets/4e757c37-0ecb-40cd-b438-7a0af107de37)
> 
> 
> 
> I use buefy in my nuxt project as nuxt plugin. Here's the code of my buefy nuxt plugin:
> 
> 
> 
> ```ts
> 
> import Buefy from 'buefy'
> 
> import 'buefy/dist/buefy.css'
> 
> 
> 
> export default defineNuxtPlugin((nuxtApp) => {
> 
>   nuxtApp.vueApp.use(Buefy, {})
> 
> })
> 
> ``` 

As far as I know there is no extension specifically for this in vscode. That honestly be something that could be very (maybe a future project idea?) helpful however the docs are always the best reference tool. From my understanding all you can do is try to rely on intelligence or volar.

**Reply by phantom-yak-venus** [reply, marked as answer]
> That honestly be something that could be very (maybe a future project idea?) helpful however the docs are always the best reference tool.

The TS Migrate Process is almost finished as far as I know. So we will have TypeScript declarations as @kikuomax mentioned [here](https://github.com/ntohq/buefy-next/issues/151#issuecomment-1835101797) therefore we will have code suggestions. So we don't need some extra project to support this feature. I was just curious if there's a way to have it before the Migrate Process is finished. Anyway, thanks for your answer 🙌
